src/lib/plugin.js

```diff
@@ -379,10 +379,11 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' is not mandatory
-    ['currency_code', 'scale', 'connectors'].map(field => {
-      if (!res.body[field]) { throwErr() }
-    })
+    // the 'ilp_prefix' field is not mandatory
+    const fields = Object.keys(res.body)
+    if (fields.indexOf('currency_code') === -1) { throwErr() }
```

Comment:
Yes, that's [what I had](https://github.com/interledgerjs/ilp-plugin-bells/pull/118/commits/0128e56e36e873b895cdd3b063e7fad6a045d9e9#diff-83847972f72b8fc383c14abb9f0791e1L383) but I was causing the integration tests to fail, like in [build 485](https://circleci.com/gh/interledgerjs/ilp-plugin-bells/485), with:

```
mark 2017-03-31T14:39:42.008Z connector:app error ExternalError: Unable to determine ledger metadata
mark     at throwErr (/home/ubuntu/ilp-plugin-bells/integration-test/node_modules/ilp-plugin-bells/src/lib/plugin.js:366:13)
mark     at map.field (/home/ubuntu/ilp-plugin-bells/integration-test/node_modules/ilp-plugin-bells/src/lib/plugin.js:384:31)
mark     at Array.map (native)
mark     at FiveBellsLedger._fetchLedgerMetadata (/home/ubuntu/ilp-plugin-bells/integration-test/node_modules/ilp-plugin-bells/src/lib/plugin.js:383:46)
mark     at next (native)
mark     at onFulfilled (/home/ubuntu/ilp-plugin-bells/integration-test/node_modules/co/index.js:65:19)
mark     at process._tickCallback (internal/process/next_tick.js:103:7)
ledger:3001 2017-03-31T14:39:42.010Z ledger:koa info --> GET / 200 6ms 804B
```

I think either there's a typo in that line, or Array.map is not supported by the node version that runs our integration tests. With this change, the integration tests get past that point, like in [build 489](https://circleci.com/gh/interledgerjs/ilp-plugin-bells/489), so I'll leave it like this for now while I fix the rest of the integration tests, and then switch it back when I find out why this is happening. :)